app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -76,6 +76,8 @@ $inplace-edit--color--very-dark: #cacaca
     & > .inplace-edit--date-range-end-date-picker
       @extend %inline-date-picker-container-position-absolute
       right: 0
+  .ui-datepicker .ui-datepicker-calendar td a.ui-state-active
+    border-color: #008bd0 !important
```

Comment:
I don't see the need for the `!important` here, works without for me.

@myabc Are there any rules we have regarding usage of this keyword?
I can only hope for "as few as possible".

@hirtie-maxim Are there special circumstances that make it neccessary?